language/move-model/src/lib.rs

```diff
@@ -265,6 +272,19 @@ fn run_spec_checker(env: &mut GlobalEnv, units: Vec<CompiledUnit>, mut eprog: Pr
     builder.warn_unused_schemas();
 }
 
+fn move_to_front(name: &str, units: &mut Vec<CompiledUnit>) {
```

Comment:
I would probably replace `name: str` to `module_id: ModuleId` here, but since this is only a temporary solution, I am OK with it as it is now.

Reply:
That was my same thought -- its only temporary and this was easier ;-)